components/datatypes/internal-site-search.schema.json

```diff
@@ -0,0 +1,155 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/datatypes/internal-site-search",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Internal Site Search",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:tags": {
```

Comment:
We talked about not having the industry tags in the data types right ?

Reply:
Correct - I've updated the next commit to reflect that, and removed it from the automated process.